src/main/java/com/microtripit/mandrillapp/lutung/model/LutungGsonUtils.java

```diff
@@ -64,7 +64,7 @@ public final Date deserialize(final JsonElement json,
 
 			if(!json.isJsonPrimitive()) {
 				throw new JsonParseException(
-						"Unexpected type for date: " +json.toString());
```

Comment:
This call to `toString()` seemed to do nothing (`toString` is called implicitly anyway)